### DISCUSSION TITLE
Xray Config Templates

### DISCUSSION THREAD

**Discussion by iamtheted** [Gozargah/Marzban#302] [Ideas]
درود بر عزیزان مرزبان
اساتید عرصه وی‌پی‌ان
ساقی‌های کانفیگ‌
امیدوارم حالتون خوب باشه💙

لطفا خودتون حتما با دستور زیر Public Key و Private Key بگیرین و جایگزین کنین
```
docker exec marzban-marzban-1 xray x25519
```
برای اینکه این دستور به ما Public Key و Private Key رو بده باید اول مرزبان رو نصب داشته باشین و Xray 1.8.1 ران شده باشه تا این دستور کار کنه.

[Vless TCP Reality.zip](https://github.com/Gozargah/Marzban/files/11619195/Vless.TCP.Reality.zip)
[Vless GRPC Reality.zip](https://github.com/Gozargah/Marzban/files/11619238/Vless.GRPC.Reality.zip)
[Vless H2 Reality.zip](https://github.com/Gozargah/Marzban/files/11619253/Vless.H2.Reality.zip)

-------------------------------------------------

این زیر براتون نمونه فالبک برای ریلیتی رو میزارم هرچند ریلیتی فالبک نداره و این نمونه فقط برای inbound اول کار میکنه یعنی اگر یک inbound دیگه مثل همین رو زیرش اضافه کنین کل کانفیگاتون از کار میفتن لذا فقط inbound اول رو برای ریلیتی میتونین فالبک بزنین و یادتون نره Public Key و Private Key خودتون رو جایگزین کنین امیدوارم به دردتون بخوره.

[Vless TCP Reality Fallback.zip](https://github.com/Gozargah/Marzban/files/11645679/Vless.TCP.Reality.Fallback.zip)
[Vless GRPC Reality Fallback.zip](https://github.com/Gozargah/Marzban/files/11645687/Vless.GRPC.Reality.Fallback.zip)
[Vless H2 Reality Fallback.zip](https://github.com/Gozargah/Marzban/files/11645692/Vless.H2.Reality.Fallback.zip)

**Comment by mahanmi** [comment]
تد عشق

**Comment by YasharMir** [comment]
دوستان منظور از فالبک چی هست در پست اول نوشته شده

**Reply by iamtheted** [reply]
درود عزیز
از فالبک برای تک پورت کردن کانفیگ ها استفاده میشه.

**Comment by YasharMir** [comment]
تک پورت کردن یعنی برای هر کاربر یک پورت مجزا تعریف میشه یا همه ی کاربران از یک پورت استفاده می کنن ؟ و سوال بعدی اینکه در حال حاضر کدوم یکی از این کانفیگ ها در همه ی اپراتور ها کار می کنن ؟

**Reply by iamtheted** [reply]
سلام عزیز
تک پورت یعنی همه یوزرها روی یک پورت باشن ، در خصوص سوال بعدیتون کار کردن کانفیگ روی یه اوپراتور به آیپی شما بستگی داره که باید تمیز باشه و SNI درست که باید ست بشه. پنل به تنهایی معجزه نمیکنه صرفا یه ابزار برای مدیریت کانفیگ هاست.

**Comment by YasharMir** [comment]
خب من بخوام از فالبک استفاده کنم دیگه از بالایی ها لارم نیست استفاده کنم درسته ؟ هر سه نوع کافیگ فالبک را می تونم در قسمت inbound ها بزارم ؟ و برعکسشم صادقه ؟ یعنی از کانفیگ های بالایی استفاده کنم دیگه نباید از فالبک ها استفاده کنم ؟ درسته ؟
